surround/remote/cli.py

```diff
@@ -67,23 +67,11 @@ def add_remote_parser(sub_parser):
     remote_parser.add_argument('-n', '--name', help="Name of the remote")
     remote_parser.add_argument('-u', '--url', help="Url of the remote")
     remote_parser.add_argument('-a', '--add', help="Used to add a remote", action='store_true')
-    remote_parser.add_argument('-t', '--type', choices=['data', 'model'])
+    remote_parser.add_argument('-t', '--type', choices=['data'])
```

Comment:
I mainly left this since I wasn't sure if I was going to use the same tool to handle the pushing and pulling for the experimentation storage, but I don't think I am going to anymore so I will remove it

Reply:
Removed 👍